hidimstat/clustered_inference.py

```diff
@@ -157,20 +157,16 @@ def clustered_inference(X_init, y, ward, n_clusters, train_size=0.3,
         Estimated parameter vector or matrix.
 
     pval : ndarray, shape (n_features,)
-        p-values, testing the negativity (low p-values means
-        that we may reject negativity).
+        One-sided p-values, low p-values characterize positive effect sizes.
 
     pval_corr : ndarray, shape (n_features,)
-        Corrected p-values, testing the negativity (low p-values means
-        that we may reject negativity).
+        Corrected p-values, low p-values characterize positive effect sizes.
 
     one_minus_pval : ndarray, shape (n_features,)
-        p-values, testing the positivity (low p-values means
-        that we may reject positivity).
+        One-sided p-values, low p-values characterize negative effect sizes.
```

Comment:
Sorry this one is still misleading.
Let me propose:
one minus the p-value, with numerically accurate values, for negative effects (ie pvalue close to one).